README.md

```diff
@@ -2,30 +2,86 @@ Core ML Community Tools
 =======================
 
 Core ML community tools contains all supporting tools for Core ML model
-conversion and validation. This includes scikit-learn, LIBSVM, Caffe,
-Keras and XGBoost.
+conversion, editing and validation. This includes scikit-learn, LIBSVM, Caffe,
+Keras, Tensorflow and XGBoost.
 
-coremltools 3.0
----------------
-[Release notes](https://github.com/apple/coremltools/releases/)
+# Table of contents
 
+* [Release information](#Release-Information)
+* [Neural network conversion guide](#Neural-network-conversion-guide)
+* [Core ML specification](#Core-ML-specification) 
+* [coremltools documentation](#coremltools-documentation-and-examples)
+* [Installation](#Installation)
 
-👍👎 Please take this quick poll and let us know how you liked this release, [here](https://github.com/apple/coremltools/blob/master/release-feedback.md)!
+
+
+## Release information
+
+Latest version of coremltools is 3.1  
 
 ```shell
 # Install using pip
-pip install coremltools==3.0
+pip install coremltools==3.1 
+# or
+pip install -U coremltools
 ```
 
-API
----
-- [Example Code Snippets](docs/APIExamples.md)
-- [coremltools Documentation](https://apple.github.io/coremltools)
-- [Core ML Specification Documentation](https://apple.github.io/coremltools/coremlspecification/)
-- [IPython Notebooks](https://github.com/apple/coremltools/tree/master/examples)
+For the latest changes please see the [release notes](https://github.com/apple/coremltools/releases/)
+
+**10/31/2019**: Released coremltools 3.1. Added support for models defined via the tf.keras API in TF 2.0. There are a few known issues, please see the release notes.   
```

Comment:
These belong in the reader notes?

Reply:
These are sort of quick peek at the release notes, so that people see it as soon as they see the readme. We'll add a line here whenever we do a release, the release notes will be more detailed. 
They won't get stale because they have dates:)

Reply:
I still strongly recommend a single source of truth. Use release notes for release information and readme for persistent information. It matches expectation for users too.

Reply:
Okay I see your point. 
In that case, here is a proposal: we start off with a  `quick installation` section in which all we do is state `pip install -U coremltools` and point to the release notes.

Reply:
Removed the release info section.